### PR TITLE
Disables Gaia mutation

### PR DESCRIPTION
**Pull request by Iamgoofball**
I like Gaia. I really do. But it's hit the point where everyone's getting so mad over botany that they're trying to remove my weed codebabby.

Weed did nothing wrong. This war over how high botany can get has gone past the line.

Just upgrade the trays with T3/T4 parts. It'll last almost as long as Gaia does, with better stats to boot.

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
this kills the botany

**Comment by evsey9** [comment]
nobody ever for any reason upgrades botany trays

**Comment by ma44** [comment]
Grudge code for kids trying to rekt weed

**Comment by Armhulen** [comment]
WAHHHHH I HAVE TO PUT IN THE BARE MINIMUM OF EFFORT FOR OVERPOWERED DEATH PLANTS

:+1:

**Comment by Tacolizard** [comment]
Are you kidding me? Grow up!

Ganja is for goons, no thanks.

Get a job, you hippie wastoid.

No thanks, I’m a good person.

You need to go to jail, hempo.

My dad told me better, no way.

Grass is crass, also gross! No!

Uhhh… no thanks loser!

Get away from me, THC addict.

Yeah right, I’m way too smart.

Let me think…No way, never.

No. You are trash if you toke.

Back off, bucko. You’re bad.

I would rather not, okay?

Injecting weed is for dummies.

I will never do one toke.

Absolutely not, I love myself.

Get a grip you sativa snorter!

Bugger off, you bong addict!

I will use my taser on you.

What do I look like? A failure?

Nah, bongs are wrong.

No way! Hemp is horrible.

I’d rather not be a cannibal.

I don’t think so, I’m nice.

I was raised right, I won’t light.

I’d like to keep my job, thanks.

You wish, pot junker! Back off!

I’m calling the Coast Guard!

No tokes for me, I’m cool.

Leave me be, you blunt blazer!

No, I’m as clean as a whistle.

That’s a death “roach”. No.

I’ll pass on your pot offer.

Cannabis is crap, you cretin!

Pish posh, pot is for the birds!

Nope. THC is not for me.

Step out of my zone, now.

Get off my case, weed stoner.

Nuh uh, I respect the police.

Lay off, I listen to the law.

NO! Blunts are for bad men.

I’d rather not die. Tokes kill.

No, weeds are for whacking.

Marijuana is for morons, ok?

Are you serious? Get a life.

You’re dumb if you do “dank”.

Stoners are loners. I’m good.

Nope! Spliffs are for wimps!

No, man. I’m straight.

**Comment by banthebantz** [comment]
gay

**Comment by Iamgoofball** [comment]
@banthebantz I know it is but people are trying to remove weed entirely over this fucking plant

**Comment by banthebantz** [comment]
then remove them, weed is cool but so is my super weeds

**Comment by Iamgoofball** [comment]
both PRs remove superweeds, the other one just removes all weed alongside it

**Comment by kevinz000** [comment]
For once I'm going to say the haters can go. If Gaia needs a Nerf nerf it don't remove.

**Comment by Repukan** [comment]
Dude weed lmao

**Comment by octareenroon91** [comment]
Fukken Gaia. Nobody ever saves their Deus seeds anymore.

I am OK with this solution.  As a possible alternative, require a large, fixed dose of earthsblood to Gaia a tray.

**Comment by FantasticFwoosh** [comment]
@Iamgoofball my PR does not remove weed, it makes starthistle (that yellow plant formerly known as /weeds) into its mutational ancestor. Gaia is one of the intended targets though, and if this PR goes through it lessens my reason to go ahead with my PR & instead atomise it.

File: code/modules/hydroponics/grown/ambrosia.dm [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	species = "ambrosiadeus"
 	plantname = "Ambrosia Deus"
 	product = /obj/item/weapon/reagent_containers/food/snacks/grown/ambrosia/deus
-	mutatelist = list(/obj/item/seeds/ambrosia/gaia)
+	mutatelist = list()
```
Comment (Really-Good-Soda-Flavor) [inline code/modules/hydroponics/grown/ambrosia.dm]: If you're going to do this, at least make it `mutatelist = null` you barbarian!

**Review by ghost — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/ambrosia.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/modules/hydroponics/grown/ambrosia.dm]: @iamgoofball

Comment (Iamgoofball) [inline code/modules/hydroponics/grown/ambrosia.dm]: Bad idea. If something assumed this was a valid list we'd have a runtime.

**Comment by KorPhaeron** [comment]
>play job
>wow it sure would be a """""quality of life"""""" change if my job did itself!
>why am I bored and why doea everyone want to nerf me??

**Pull request merged**